src/lib/components/Header/Nav/Nav.svelte

```diff
@@ -0,0 +1,26 @@
+<script lang="ts">
+  import type { NavItemType } from "./types";
+
+  import NavItem from "./NavItem.svelte";
+
+  export let items: NavItemType[] = [];
+
+  // The index of the item that is currently expanded.
+  let expandedI: number | undefined = undefined;
```

Comment:
nit (i.e., change at your discretion): `expandedIndex` is more explicit and removes the need for a comment

Reply:
Agreed, I'll update this.